### PR TITLE
add tensorrt-llm to inference

### PR DESCRIPTION
**Pull request by Jerry-Kon**
/kind documentation

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -69,6 +69,8 @@
...
 | **[vLLM](https://github.com/vllm-project/vllm)** | ![Stars](https://img.shields.io/github/stars/vllm-project/vllm.svg) | ![Release](https://img.shields.io/github/release/vllm-project/vllm) | ![Contributors](https://img.shields.io/github/contributors/vllm-project/vllm) | A high-throughput and memory-efficient inference and serving engine for LLMs | |
+| **[TensorRT-LLM](https://github.com/NVIDIA/TensorRT-LLM)** | ![GitHub Repo stars](https://img.shields.io/github/stars/NVIDIA/TensorRT-LLM) | ![GitHub Release](https://img.shields.io/github/v/release/NVIDIA/TensorRT-LLM) | ![GitHub contributors](https://img.shields.io/github/contributors/NVIDIA/TensorRT-LLM) | TensorRT-LLM provides users with an easy-to-use Python API to define Large Language Models (LLMs) and build TensorRT engines that contain state-of-the-art optimizations to perform inference efficiently on NVIDIA GPUs.||
```
Comment (kerthcet) [inline README.md:72]: We put the names in the alphabetical order, so please reorder this, thanks!

**Review by kerthcet — COMMENTED** [review]
Please squash the same time.

**Comment by kerthcet** [comment]
Kindly ping @Jerry-Kon

**Comment by Jerry-Kon** [comment]
complete the merge requirement

**Comment by kerthcet** [comment]
could you squash the commits?

**Comment by Jerry-Kon** [comment]
squash the history commit

**Comment by kerthcet** [comment]
/lgtm

Thanks!

**Pull request merged**